src/lib/features/instance-stats/instance-stats-service.ts

```diff
@@ -219,6 +219,22 @@ export class InstanceStatsService {
         return settings?.enabled || false;
     }
 
+    async hasPasswordAuth(): Promise<boolean> {
+        const settings = await this.settingStore.get<{ disabled: boolean }>(
+            'unleash.auth.simple',
+        );
+
+        return settings?.disabled !== false;
```

Comment:
password auth has different logic than others (disable instead of enabled check)